packages/accordion/__tests__/accordion.test.tsx

```diff
@@ -36,9 +36,86 @@ describe("<Accordion />", () => {
       });
       expect(results).toHaveNoViolations();
     });
+
+    it("accepts a custom ID", () => {
+      let { container } = render(
+        <Accordion id="test-id">
+          <AccordionItem>
+            <AccordionButton>Button One</AccordionButton>
+            <AccordionPanel>Panel One</AccordionPanel>
+          </AccordionItem>
+          <AccordionItem>
+            <AccordionButton>Button Two</AccordionButton>
+            <AccordionPanel>Panel Two</AccordionPanel>
+          </AccordionItem>
+        </Accordion>
+      );
+
+      expect(container.querySelector("#test-id")).toHaveAttribute(
```

Comment:
I can't think of a way to test this without querySelector, any ideas?

Reply:
I'd probably use `getByTestId` here, or just assert that the element exists